File: submissions/sm_014_kondreddy/week_02/day_03/session_02/count_consonants.js
var string="masaiSchool";
var i;
var j;
var vowels = ['a','e','i','o','u']
count = 0;
for (i = 0;i < string.length;i++) {
    for(j = 0;j < vowels.length;j++) {
    if (string[i] == vowels[j]) {
        count = count+1;
        // console.log(string[i])
        }
    }
}
console.log(string.length-count)

File: submissions/sm_103_apoorva/week_13/day_4/count_consonants.py
userInput = list(input("Enter values of list with space between: ").split())
print(userInput)
vowels = ["A","E","I","O","U","a","e","i","o","u"]
count = 0
for i in userInput:
    for j in i:
        if j not in vowels:
            count += 1
print(count)